app/src/main/java/com/framgia/forder/screen/shopmanagement/ListShopManagementAdapter.java

```diff
@@ -27,6 +27,7 @@
     ListShopManagementAdapter(@NonNull Context context) {
         this.mContext = context;
         mShopManagements = new ArrayList<>();
+        ArrayList<ShopInfo> shopInfos = new ArrayList<>();
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to shopInfos in new com.framgia.forder.screen.shopmanagement.ListShopManagementAdapter(Context)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE